src/electionguard/rsa.py

```diff
@@ -0,0 +1,89 @@
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.asymmetric.rsa import (
+    generate_private_key,
+    RSAPrivateKey,
+    RSAPrivateKeyWithSerialization,
+    RSAPublicKey,
+)
+from cryptography.hazmat.primitives.asymmetric.padding import PKCS1v15
+from cryptography.hazmat.primitives.serialization import (
+    Encoding,
+    load_pem_private_key,
+    load_pem_public_key,
+    NoEncryption,
+    PrivateFormat,
+    PublicFormat,
+)
+from typing import NamedTuple
+
+
+PUBLIC_EXPONENT = 65537
+KEY_SIZE = 4096
```

Comment:
🚨🚨🚨🚨🚨ALERT🚨🚨🚨🚨🚨🚨

This seems to be related to the error in the build. As far as I can tell, there are test cases which exceed the 4096 bit character and the error is consistent.

Reply:
I have tried up to 8000 so I'm not quite sure where to go from here.